enhancements/monitoring/restart-metrics.md

```diff
@@ -0,0 +1,105 @@
+---
+title: restart-metrics
+authors:
+  - "@rphillips"
+reviewers:
+  - "@smarterclayton"
+  - "@sjenning"
+  - "@mrunalp"
+approvers:
+  - TBD
+creation-date: 2020-03-23
+last-updated: 2020-03-23
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Service Restart Metrics
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+There is not any insight with systemd units restarting. Metrics of this sort are
```

Comment:
So we only want to keep metrics about systemd units restarting for core kube and openshift components e.g. crio, kubelet? And drop the rest of the metrics?

Reply:
openshift components are bit more complex, since they usually run in a container. This proposal is only for components that run on the host (not in a container): kubelet, crio are the standouts

Reply:
I think we want restart components for everything on the host.  All host .service restarts should be monitored.  We own those too.

Reply:
Guessing thats the reason why you are not going to extract the start time from the go process collection metrics, as that would not work for all components? `process_start_time_seconds`

Reply:
@lilic correct